src/app/client/src/app/modules/core/components/main-footer/main-footer.component.html

```diff
@@ -51,7 +51,7 @@
             </div>
             <label
               class="d-block text-center font-weight-bold">{{resourceService?.frmelmnts?.lbl?.downloadDikshaForMobile | interpolate:'{instance}': instance}}</label>
-            <a class="d-block app-download" (click)="redirectToMobileApp()" tabindex="0">
+            <a class="d-block app-download" (click)="redirectToMobileApp()" href="javascript:void(0);" tabindex="0">
```

Comment:
you cannot give href to the a tag

Reply:
@rajeevsathish  converted anchor tag to span.Kindly review and merge